src/main/java/org/jboss/xavier/integrations/jpa/service/AnalysisService.java

```diff
@@ -17,6 +17,12 @@
 @Component
 public class AnalysisService
 {
+    public enum STATUS {
```

Comment:
- I'd suggest to move the STATUS enum to his own file and locate it into the package `org.jboss.xavier.analytics.pojo.output`
- I suggest also to change the AnalysisModel#status to: `@Enumerated(EnumType.STRING) private String status`. In this way, we will avoid using `STATUS.IN_PROGRESS.toString()` and we can use directly the ENUM value

Reply:
Honestly, I dont see which is the benefit. We would avoid using the toString but we should implement a class and a method.....
Do you see any harm on having it like this ?

Reply:
https://jira.coreos.com/browse/MIGENG-214